Rubberduck.Core/Settings/HotkeySetting.cs

```diff
@@ -29,14 +29,7 @@ public class HotkeySetting
         [XmlIgnore]
         public string Prompt => RubberduckUI.ResourceManager.GetString($"CommandDescription_{CommandTypeName}", CultureInfo.CurrentUICulture);
 
-        public override string ToString()
-        {
-            return string.Format("{0}{1}{2}{3}",
-                HasCtrlModifier ? KeyModifierCtrl : string.Empty,
-                HasShiftModifier ? KeyModifierShift : string.Empty,
-                HasAltModifier ? KeyModifierAlt : string.Empty,
-                Key1);
-        }
+        public override string ToString() => $"{(HasCtrlModifier ? KeyModifierCtrl : string.Empty)}{(HasShiftModifier ? KeyModifierShift : string.Empty)}{(HasAltModifier ? KeyModifierAlt : string.Empty)}{Key1}";
```

Comment:
what's up with this? seems less readable?

Reply:
Yeah, agreed. will revert.

Reply:
Done.